1.9/administration/installing/custom/configuration-parameters.md

```diff
@@ -20,6 +20,36 @@ This parameter specifies a YAML nested list (`-`) of IPv4 addresses to your [pri
 ### bootstrap_url
 This required parameter specifies the URI path for the DC/OS installer to store the customized DC/OS build files. If you are using the automated DC/OS installer, you should specify `bootstrap_url: file:///opt/dcos_install_tmp` unless you have moved the installer assets. By default the automated DC/OS installer places the build files in `file:///opt/dcos_install_tmp`.
 
+### cluster_docker_credentials
+This parameter specifies a dictionary of Docker credentials to pass. 
+
+- If unset, the default is an empty credentials file is placed in the
+`/etc/mesosphere/docker_credentials` during DC/OS install. You can then use sysadmin scripts to edit the file and do a `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public`.
+- You can also set as the `--docker_config` JSON [format](http://mesos.apache.org/documentation/latest/configuration/). In the `config.yaml` file you can write YAML and it will automatically be mapped to the JSON format for you. This will store the Docker credentials in the same location as the DC/OS internal configuration (`/opt/mesosphere`). If you need to update or change the configuration, you will have to create a new DC/OS internal configuration.
+
+You can use the following options to further configure the Docker credentials:
+
+*  **cluster_docker_credentials_dcos_owned** This parameter specifies whether to store the credentials file in `/opt/mesosphere` or `/etc/mesosphere/docker_credentials`.
+
+    *  `cluster_docker_credentials_dcos_owned: 'true'` The credentials file is stored in `/opt/mesosphere`.
+    
+        *  **cluster_docker_credentials_write_to_etc** This parameter specifies whether to write a cluster credentials file.
+        
+            *  `cluster_docker_credentials_write_to_etc: 'true'` Do not write a credentials file. This can be useful if overwriting your credentials file will cause problems (e.g. if it is part of a machine image or AMI). This is the default value.
+            *  `cluster_docker_credentials_write_to_etc: 'false'` Write a credentials file.
+            
+    *  `cluster_docker_credentials_dcos_owned: 'false'` The credentials file is stored in `/etc/mesosphere/docker_credentials`.
+
+*  **cluster_docker_credentials_enabled** This parameter specifies whether to pass the Mesos `--docker_config` option to Mesos.
+
+    *  `cluster_docker_credentials_enabled: 'true'` Pass the Mesos `--docker_config` option to Mesos.
```

Comment:
Pass the Mesos `--docker_config` option to Mesos. It will be set to point at a file containing the provided `cluster_docker_credentials` data.